src/java/org/apache/cassandra/tools/NodeProbe.java

```diff
@@ -126,6 +128,11 @@ public class NodeProbe implements AutoCloseable
      * @throws IOException on connection failures
      */
     public NodeProbe(String host, int port, String username, String password) throws IOException
+    {
+        this(host, port, username, password, System.out);
+    }
+
+    public NodeProbe(String host, int port, String username, String password, PrintStream output) throws IOException
```

Comment:
Can we avoid breaking the constructor API here? The oss patch gets around this with the `setOutput` method
https://github.com/apache/cassandra/commit/83e1e9e45193322f18f57aa7cc4ad31d9d5a152d#diff-92f8748902f03f37a4f7db56b4dfb7d226adcf3839141e6adb8ebbc575020d57R277